website/source/docs/platform/k8s/helm.html.md

```diff
@@ -443,84 +443,50 @@ consul-server-2                            10.60.2.197:8301  alive   server  1.4
 
 ## Helm Chart Examples
 
-The below values.yaml can be used to set up a single server Consul cluster with a LoadBalancer to allow external access to the UI and API.
+The below `values.yaml` can be used to set up a single server Consul cluster with a `LoadBalancer` to allow external access to the UI and API.
 
-```
+```yaml
 global:
   enabled: true
-  image: "consul:1.4.2"
-  domain: consul
-  datacenter: dc1
 
 server:
-  enabled: true
   replicas: 1
   bootstrapExpect: 1
-  storage: 10Gi
-
-client:
-  enabled: true
-
-dns:
-  enabled: true
 
 ui:
-  enabled: true
   service:
-    enabled: true
     type: LoadBalancer
 ```
 
-The below values.yaml can be used to set up a three server Consul Enterprise cluster with 100GB of storage and automatic Connect injection for annotated pods in the "my-app" namespace.
+The below `values.yaml` can be used to set up a three server Consul Enterprise cluster with 100GB of storage and automatic Connect injection for annotated pods in the "my-app" namespace.
 
 Note, this would require a secret that contains the enterprise license key.
 
-```
+```yaml
 global:
-  enabled: true
-  domain: consul
   image: "hashicorp/consul-enterprise:1.4.2-ent"
   datacenter: dc1
 
 server:
-  enabled: true
   replicas: 3
   bootstrapExpect: 3
   enterpriseLicense:
     secretName: "consul-license"
     secretKey: "key"
   storage: 100Gi
   connect: true
-  affinity: |
```

Comment:
It's the default though https://github.com/hashicorp/consul-helm/blob/master/values.yaml#L122

Reply:
These examples are set up so that someone could copy them into a values file and run with this setup. If it isn't in the values file when they copy it over, it's not going to be applied.

Reply:
Hmm, you're right. That's not how we should recommend doing this though. A separate values.yaml file should be created and checked into source control. They shouldn't be overriding the default values.yaml. That's where we set all the defaults.

Reply:
See https://helm.sh/docs/using_helm/#customizing-the-chart-before-installing for the recommended way to customize a chart.

Reply:
None of the education resources are currently set up that way. If this is a switch we expect to be made in how we recommend doing this, there are a whole lot of other places that need to get modified (and a bunch of recorded demos and talks that won't align). We should leave these examples as-is for now, and take a look at it holistically across everything, with feedback from @judithpatudith @kaitlincarter-hc and @nicholasjackson .

Reply:
Okay yeah we shouldn't merge this yet but that's the way we should be customizing helm charts and as far as I was aware the way everyone else does it.

That makes it a lot easier for us to write the charts too because we can set our defaults in values.yaml. I'm not really sure what the behaviour of things is if we don't do it that way.

Reply:
This guide (https://learn.hashicorp.com/consul/getting-started-k8s/minikube) does it with a separate values.yaml file:
![image](https://user-images.githubusercontent.com/1034429/61821384-fd306480-ae4e-11e9-8dd0-bf7dc1ae2ae8.png)

and so does this one https://learn.hashicorp.com/consul/getting-started-k8s/helm-deploy
![image](https://user-images.githubusercontent.com/1034429/61821447-25b85e80-ae4f-11e9-869a-d3ecc7700499.png)

And so does https://learn.hashicorp.com/consul/getting-started-k8s/azure-k8s
![image](https://user-images.githubusercontent.com/1034429/61821483-3ff23c80-ae4f-11e9-8095-2ae09d88a0df.png)

@judithpatudith @kaitlincarter-hc and @nicholasjackson am I missing any other places where we might be instructing users to edit the `values.yaml` file contained in the actual chart directly rather than creating our own file and passing it to helm via the `-f` flag?